dist/keen-ui.css

```diff
@@ -1,6 +1,6 @@
 /*!
```

Comment:
Remove the dist file changes, we'll add those when we publish the release. Also remove the `lib/` file changes.